proposals/solid-oidc/index.bs

```diff
@@ -181,9 +181,9 @@ SHOULD use a WebID as their Client ID.
 
 ## WebID ## {#clientids-webid}
 
-The WebID itself MUST resolve to an RDF document that MUST include a single `solid:oidcRegistration`
-property. That property MUST be a JSON serialization of an OIDC client registration, per the definition
-of client registration metadata from [[!RFC7591]] section 2.
+The WebID itself MUST resolve to an RDF document. That document SHOULD be available as JSON-LD with an `@context`
+that includes `https://www.w3.org/ns/solid/oidc-context.jsonld` but which otherwise produces a JSON serialization of an OIDC
+client registration, per the definition of client registration metadata from [[!RFC7591]] section 2.
```

Comment:
re MUST and SHOULD use, unclear to me what's required for minimal interop. Is the intention along the lines of "a JSON-LD representation of the WebID profile document uses `@context`..."?

Would JSON-LD expanded form (without the context) be conforming? profile URI required in Content-Type?

Reply:
JSON-LD expanded form as well as Turtle would be acceptable (i.e. "MUST resolve to an RDF document")

The use of this `@context` is a recommendation because it allows an implementation to perform a pure JSON parse of the document. But if that fails, an implementation would need to do a full RDF parse, which is why any supported RDF syntax would be acceptable for minimal interop.

Reply:
I interpret "MUST" as minimal for interop:

Publisher may only provide a representation in RDFa. Another publisher only in TriG. Both conforming. Consumer must be able to parse all of the RDF syntaxes (of today) in order to interop with all publishers or it can only with some or none. Alternatively, both the publisher and the consumer can work with all of the defined RDF syntaxes, which would interop today, but may not when new syntaxes are available in the future (unless all implementations are expected to be maintained..)

The "SHOULD be available as JSON-LD" is not as significant as the MUST especially if an implementation fulfils the requirement with another syntax. I can see it working with "MUST be available as JSON-LD" as minimal for interop in addition to the possibility of consumers being able to negotiate for another syntax (re "MUST resolve to an RDF document").